### ISSUE TITLE
SaveToDb error when running ahdb.go

### ISSUE THREAD

**Issue by y2mq37** [mooreatv/AHDBapp#8]
I'm receiving the following error when attempting to process AuctionDB.lua:

> user@user-VirtualBox:~$ sudo ~/go/bin/AHDBapp < /home/user/Documents/AHDB/AuctionDB.lua
> 
> 19:33:45 F ahdb.go:211> Can't count items: Error 1045: Access denied for user 'root'@'localhost' (using password: NO)
> panic: aborting...

I'm running an Ubuntu VM, LAMP is installed, MYSQL is accessible, ahdb.go lines 285 and 286 are edited to root as the user, and my sql password. ahdb.go is also set to 775 permissions, and I confirmed that MYSQL is running via tcp 3306. The error still occurs even when I run the argument via sudo. Not sure where the hangup is..

**Comment by y2mq37** [comment]
I think I figured it out. When running ~/go/bin/AHDBapp < ...path_to_.../SavedVariables/AuctionDB.lua, the AHDBapp executable defaults the user to root, but it doesn't set the password. It's assuming that root has full authentication with MySQL, (including access without a password, or with the password set to, "MYSQL_PASSWORD", as outlined in the instructions. I'm not sure if there are additional arguments that can be added in order to query for a password prior to running the app. So I opted to work backwards, (via the older instructions):

What I ended up doing is:

1. Stop the MySQL service, and start MySQL in safe mode
2. Edit root in the mysql user table to have access without a password
3. Restart MySQL, and run `go run ahdb.go < AuctionDB.lua`

The file ran flawlessly, and imported the the data from the lua into the ahdb database.
